sls/src/org/scala/abusers/sls/BspStateManager.scala

```diff
@@ -47,24 +47,26 @@ object BspStateManager {
   */
 class BspStateManager(
     val bspServer: BuildServer,
-    sourcesToTargets: MapRef[IO, URI, Option[ScalaBuildTargetInformation]],
+    sourcesToTargets: AtomicCell[IO, Map[URI, ScalaBuildTargetInformation]],
```

Comment:
regarding the comment for this class:
IMO it would be nice to have a designated lsp method to switch between configured scala versions